src/lib.rs

```diff
@@ -1183,6 +1214,8 @@ impl Default for BindgenOptions {
             objc_extern_crate: false,
             enable_mangling: true,
             prepend_enum_name: true,
+            format_bindings: true,
```

Comment:
And this is defaulting to `true`, but you just said you changed it to default to `false` (which it should) -- did you forget to push some local changes?